### PR TITLE
remove_deprecated_for_portal-kernel_part_Eleven

### PR DESCRIPTION
**Pull request by jeffwu0724**
Hi Tina, Shuyang,

This is remove_deprecated_for_portal_kernel_part_Eleven.
can also see https://github.com/jeffwu0724/liferay-portal/pull/58 and https://github.com/liferay-core-infra/liferay-portal/pull/676
Thanks for checking.

### CODE REVIEW

**Comment by jeffwu0724** [comment]
@catarineaguiar Can you help me look at the test result, please?

Thanks
Jeff

File: portal-kernel/test/unit/com/liferay/portal/kernel/util/TextFormatterTest.java [hunk 1]
```diff
@@ -136,39 +138,39 @@ public void testformatStorageSizeOneB() throws Exception {
 		long bytes = 1;
 
 		Assert.assertEquals(
-			"1 B", TextFormatter.formatStorageSize(bytes, LocaleUtil.SPAIN));
+			"1 B", LanguageUtil.formatStorageSize(bytes, LocaleUtil.SPAIN));
```
Comment (tinatian) [inline portal-kernel/test/unit/com/liferay/portal/kernel/util/TextFormatterTest.java:141]: @jeffwu0724 The logic had been removed from TextFormatterTest, thus you do not need to test the logic here, you have two options here:
1. Move the test logic to LanguageUtilTest
2. If corresponding test logic had been tested in LanguageUtilTest, then remove it directly.

**Comment by tinatian** [comment]
@jeffwu0724 I am going to close this pull, please resend.

Thanks.
Tina.

**Comment by tinatian** [comment]
ci:close

**Pull request closed without merging**